### PR TITLE
Remove redundant text from clock icon in system tray

### LINKED ISSUE #4364: double announcement of clock icon in the system tray
**Reported by blindbhavya on 2014-08-03 11:24**
Steps to reproduce
Press Windows + B to go to the System Tray.
Press left arrow (on my machine a single left arrow is enough but maybe on other machines navigating to the cock icon may differ).
Actual Result = clock cock other information
Expected Result = clock other information
Why does NVDA read clock clock twice?

### PR DESCRIPTION
**Pull request by lukaszgo1**
First of all thanks to @dkager for providing sample implementation.
### Link to issue number:
Fixes #4364
### Summary of the issue:
On some versions of Windows name of the clock icon in system tray contain information's present in the role or in the value.
### Description of how this pull request fixes the issue:
If value is not empty we are dismissing the name.
### Testing performed:
Tested the following

| Version of Windows | before the change | after the change |
| --- | --- | --- |
| Windows 7 | Clock  clock  Time and date | clock time and date |
| Windows 8 and Server 2012 | Time and date  clock  time and date | clock time and date |
| Windows 8.1 | Time and date  clock  time and date | clock time and date |
| Windows 10 RTM (10240) | Time and date  clock  time and date | clock time and date |
| Windows 10 TH2 (10586) | Time and date  clock  time and date | clock time and date |

On Windows 10 Anniversary Update and later there is no double text, so i've just made sure, that it works as it should.
### Known issues with pull request:
None known
### Change log entry:

Section:  Bug fixes
NVDA will no longer  present redundant information when reading clock system tray on some versions of Windows.

### CODE REVIEW

**Review by dkager — DISMISSED** [review]
The code looks good. The two things to test are:
1. Does this fix not affect other Windows versions?
2. Are there still LTR/RTL characters in the clock in other Windows versions? It would be nice if this PR could address all supported Windows versions at once.

**Comment by lukaszgo1** [comment]
@dkager This PR addresses only original problem reported in #4364 and not deals with RTL character in the clock text. I've excluded later versions of Windows from a PR description, because there is no redundant information’s in the clock icon under them. My preference would be to wait until it is merged and then create additional PR which will fix #8361

**Comment by lukaszgo1** [comment]
@dkager  If i have your approval can you please request a review from @michaelDCurran or @feerenrut. I'd do it my self, but i am not a  collaborator. I don't expect it to be included in 2018.3, but having it in master would let me work on #8361 more easily.

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**
